CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Kiwix JS Contribution Guide
+
+If you have some development experience with HTML and JavaScript, we welcome Pull Requests on existing issues. Please look at this repository's Issue
+tracker, in particular those marked "good first issue". If you have found a bug, check first that we do not have an existing issue covering it, and if not, feel free to create an issue.
+
+Please follow these guidelines when contributing:
+
+- Ask to be assigned to an issue you wish to work on first (we have lots of back issues, some of which are no longer relevant or wanted);
```

Comment:
@gaurav7019 Please change "back issues" to "past issues" (change requested by mossroy on the other PR).